src/ui/session/ViewSession.tsx

```diff
@@ -3,44 +3,49 @@ import { useNavigate } from 'react-router-dom';
 import { Screens } from '@/ui/App';
 import { SubjectType } from '@/model/Subject';
 import { Button } from '@wordpress/components';
+import { getSchemas } from '@/model/Schema';
+
+const schemas = getSchemas();
 
 export function ViewSession() {
 	const { session } = useSessionContext();
 	const navigate = useNavigate();
 
+	const importButtons: { text: string; url: string }[] = [];
+	Object.keys( schemas ).forEach( ( subjectType ) => {
+		// Pages get a specific button.
+		if ( subjectType === SubjectType.Page ) {
+			importButtons.push( {
+				text: `Import Pages`,
+				url: Screens.importPagesStart( session.id ),
+			} );
+			return;
+		}
+
+		const schema = schemas[ subjectType ];
+		importButtons.push( {
+			text: `Import ${ schema.title }s`,
```

Comment:
I opted for keeping the plural like this, we can revisit this when we tackle #158.